bin/echo/echo.1

```diff
@@ -92,7 +92,7 @@ manual page.
 .Sh EXAMPLES
 Special treatment of options and backslashes:
 .Bd -literal -offset indent
-$ /bin/echo "-hello\\tworld"
+$ /bin/echo "-hello\etworld"
```

Comment:
Just to elaborate on the "maintainability" thing... anybody looking at the source code for `echo.1` will recognize the previous text.  "Ok, it says `\\tworld`, which is clearly a `\t` tab, but with the backslashes escaped."

Whereas the "correct" source is `\etworld`.  Which to my eyes, looks like "some kind of special `\e` command, followed by `tworld`.  What on earth does `tworld` mean?  That must be a typo, surely!"